cmd/cli/handler_jwk.go

```diff
@@ -123,10 +123,7 @@ func (h *JWKHandler) ImportKeys(cmd *cobra.Command, args []string) {
 		cmdx.Fatalf("Expected status code 200 or 404 but got %d while fetching data from %s", response.StatusCode, u)
 	}
 
-	var set jose.JSONWebKeySet
-	err = json.NewDecoder(response.Body).Decode(&set)
-	cmdx.Must(err, "Unable to decode payload to JSON: %s", err)
-
+	set := jose.JSONWebKeySet{}
 	for _, path := range args[1:] {
```

Comment:
Instead of ignoring the existing keys we should make sure that we update any keys that have a matching key id :)

Reply:
Hey, I updated the branch and added an update function in the importer. Existing keys (same KeyID and SID) will be replaced and new keys (new KeyID and same SID) will be added to the list.
I also updated the test to test the changes.